javascripts/discourse/components/category-featured-images.js

```diff
@@ -7,7 +7,10 @@ export default class CategoryFeaturedImages extends Component {
   @controller application;
 
   get shouldShow() {
-    return this.router.currentRoute.name.includes("discovery.category");
+    if (this.router.currentRoute.name.includes("discovery.category")) {
+      this.application.showFooter = true;
```

Comment:
I know it was already being done in the other getter but they shouldn't have side-effects, i.e. setting state, especially external to the component. I'm approving for now. I have a suggestion how to make it better but it involves multiple files so PR UI isn't the best place for that 😃